docs/03-03-examples-alpha.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Kyma CLI alpha command usage examples
+type: Details
+---
+
+The following examples show how to provision a cluster, deploy Kyma, and <!--- update is not among the alpha commands ---> with alpha commands.
+
+## Provision a cluster locally or using cloud providers
+<!---provision gke is not among the alpha commands--->
+To provision a cluster on a specific cloud provider (in this example GCP), run:
+
+```bash
+kyma alpha provision gke --credentials {SERVICE_ACCOUNT_KEY_FILE_PATH} --name {CLUSTER_NAME} --project {GCP_PROJECT} 
+```
+
+To provision a Minikube cluster, run:
+<!---provision minikube is not among the alpha commands--->
+```bash
+kyma alpha provision minikube
+```
+
+## Deploy Kyma
+
+To deploy Kyma using your own domain, run:
+
+```bash
+kyma alpha deploy --domain {DOMAIN} --tls-cert {TLS_CERT} --tls-key {TLS_KEY}
+```
+
+`{TLS_CERT}` and `{TLS_KEY}` are the TLS certificate and TLS key for the domain used. Note that both of them must be encoded in base64. You can use [OpenSSL](https://www.openssl.org/) (which is installed by default with macOS) to convert your TLS certificate and TLS key to base64-encoded values.
```

Comment:
The arguments changed. The cert and key must be passed as a file now. Please provide a full example with a self-signed certificate and a fake domain, then people can try it out

Reply:
@NHingerl : users can create a self-signed certificate by running this command:
`openssl req -x509 -newkey rsa:4096 -keyout key.pem -out crt.pem -days 365`

Reply:
I assume creating the self-signed cert is a prerequisite to be done before running the `alpha deploy` command, right?

I'm not sure what to put in a full example. Pls provide sample text.